docs/gettingstarted/buildingtest.rst

```diff
@@ -332,6 +333,38 @@ If you try to exceed this bound you will get an error such as
     .. command-output:: buildtest build -b tutorials/test_status/pass_returncode.yml --rebuild 51
         :returncode: 1
 
+Limit Number of Tests
+----------------------
+
+The `buildtest build` command can limit the number of tests that can run via ``--limit`` option. This
+can be useful when running large number of tests and you have no idea
+how many tests will run. The ``--limit <NUM>`` option expects a positive number which will
+limit number of tests to the total limit. If there are less tests to run than the
+value specified by ``--limit``, then buildtest will run all the test. When buildtest has more
+tests to run than the value specified by ``--limitt``, then buildtest will exclude some tests.
```

Comment:
Spelling error ``limit`` instead of ``limitt``.

Reply:
thanks for the catch, i fixed this